Documentation/concepts/networking/masquerading.rst

```diff
@@ -52,9 +52,90 @@ Implementation Modes
 --------------------
 
 eBPF-based
-  The eBPF-based implementation is the most efficient
-  implementation. It requires Linux kernel 4.19 and can be enabled with the
-  option ``enable-bpf-masquerade: true``.
+**********
+
+The eBPF-based implementation is the most efficient
+implementation. It requires Linux kernel 4.19 and can be enabled with
+the ``global.bpfMasquerade=true`` helm option (enabled by default).
```

Comment:
Note that the previous section describes from a high-level how to set the masquerade interface, and it implies that iptables is required to configure masquerading interfaces from the sentence "This setting is only available in....". I think we should (even if briefly) expand the above section to point out that there is a BPF-based implementation as well which is configured the devices flag.

Reply:
That's a good spot. As configuring the masquerading interface depends on the implementation, I moved the `--egress-masquerade-interface` bits to the `iptables-based` section instead.